idaes/models_extra/column_models/tests/test_MEAsolvent_column.py

```diff
@@ -0,0 +1,182 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Tests for MEA solvent column model
+Author: Anuja Deshpande, Andrew Lee
+"""
+# Import Python libraries
+import pytest
+
+# Import Pyomo libraries
+from pyomo.environ import (ConcreteModel,
+                           value,
+                           Constraint,
+                           check_optimal_termination,
+                           units as pyunits)
+
+# Import IDAES Libraries
+from idaes.core import FlowsheetBlock
+from idaes.models_extra.column_models.solvent_column \
+    import PackedColumn
+from idaes.models_extra.column_models.MEAsolvent_column import MEAColumn
+from idaes.models.properties.modular_properties.base.generic_property import (
+        GenericParameterBlock)
+from idaes.models_extra.column_models.properties.MEA_vapor \
+    import flue_gas as vaporconfig_absorber
+from idaes.models_extra.column_models.properties.MEA_solvent \
+    import configuration as liquidconfig
+
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+from idaes.core.util import get_solver
+
+import idaes.logger as idaeslog
+
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_solver()
+solver.options["bound_push"] = 1e-10
+
+class TestAbsorber:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+    
+        # Set up molar flow bounds
+        vaporconfig_absorber["state_bounds"]["flow_mol"] = (0, 1, 1e6, pyunits.mol / pyunits.s)
```

Comment:
You should create a copy of the config dict before doing this - otherwise you have just changed the original version which might affect other tests which are run later.